### PR TITLE
Refactoring #276

### LINKED ISSUE #276: Refactor CodonDataConstants.$block1
I've selected [**CodonDataConstants.$block1**](https://github.com/Verisimilitude11/DNAnalyzer/blob/36f7bad03e75bcbcdca7a38852c48754bb27b149/src/main/java/DNAnalyzer/codon/CodonDataConstants.java#L68-L91) for refactoring, which is a unit of **24** lines of code. Addressing this will make our codebase more maintainable and improve [Better Code Hub](https://bettercodehub.com)'s **Write Short Units of Code** guideline rating! 👍 

Here's the gist of this guideline:
- **Definition** 📖 
Limit the length of code units to 15 lines of code.
- **Why**❓ 
Small units are easier to analyse, test and reuse.
- **How** 🔧 
When writing new units, don't let them grow above 15 lines of code. When a unit grows beyond this, split it in smaller units of no longer than 15 lines.

You can find more info about this guideline in [Building Maintainable Software](http://shop.oreilly.com/product/0636920049159.do). 📖 
 
---- 
ℹ️ To know how many _other_ refactoring candidates need addressing to get a guideline compliant, select some by clicking on the 🔲  next to them. The risk profile below the candidates signals (✅) when it's enough! 🏁 

----
Good luck and happy coding! :shipit: :sparkles: :100:

### PR DESCRIPTION
**Pull request by salahhalkhoums**
I open this PR as part of Strengthening the  codebase of DNAnalyzer.

Closes #276.

### CODE REVIEW

**Review by LimesKey — COMMENTED** [review]
Looks good to me, thanks for your contribution. I'll just wait for another person to do the code review fully.

**Comment by Verisimilitude11** [comment]
@salahhalkhoums could you please clear the merge conflicts? Thanks!

**Review by ImpossibleReality — DISMISSED** [review]

File: src/main/java/DNAnalyzer/codon/CodonDataConstants.java
Comment (Verisimilitude11) [reply src/main/java/DNAnalyzer/codon/CodonDataConstants.java]: Thanks for noticing this @ImpossibleReality, I hadn't noticed that the Gradle build failed. I'll merge after some testing later today @salahhalkhoums. By the way, do you use GitHub Codespaces or JetBrains Space?

**Review by Verisimilitude11 — COMMENTED** [review]
Looks like the code is able to compile now, I'll get back to you in about 4 hours with a response @salahhalkhoums. Thanks for contributing!

**Review by ImpossibleReality — COMMENTED** [review]
Thanks for fixing! Can't review right now, but might be able to later.

**Review by Verisimilitude11 — APPROVED** [review]

**Review by Verisimilitude11 — APPROVED** [review]

**Pull request merged**
